strawberryfields/backends/bosonicbackend/backend.py

```diff
@@ -15,43 +15,42 @@
 """Bosonic backend"""
 import warnings
 
-from numpy import (
-    empty,
-    concatenate,
-    arange,
-    array,
-    identity,
-    arctan2,
-    angle,
-    sqrt,
-    vstack,
-    zeros_like,
-    allclose,
-    ix_,
-    zeros,
-    shape,
-)
+import numpy as np
+from scipy.linalg import block_diag
+
 from thewalrus.samples import hafnian_sample_state, torontonian_sample_state
```

Comment:
I think we can remove this import.

Reply:
You mean block_diag from scipy?

Reply:
I meant the walrus imports, bu then also need to update a few methods.

Reply:
Will do this in a subsequent commit!